src/apps/bridge/sensor_drivers/borealisSensor.cpp

```diff
@@ -608,6 +714,10 @@ Borealis_t *createBorealisSensorSub(uint64_t node_id) {
   static constexpr uint32_t default_borealis_reading_period_ms = 1000;
   static constexpr char reading_period_key[] = "bandsSampleTimeMs";
   static constexpr char level_statistics_period_key[] = "report_interval";
+  static constexpr char hydrotwin_ldr_key[] = "hydrotwin_ldr";
+  static constexpr char hydrotwin_hdr_key[] = "hydrotwin_hdr";
+  static constexpr uint32_t hydrotwin_ldr_minute_default = 5;
```

Comment:
The typical default we plan to ship single-BOREALIS systems with is `12` for the LDR minutes. I'm not 100% sure we should change this particular value. The goal would be to ensure that the hydrotwin LDR config is at least 90 seconds shorter than the bridge sample duration.